src/components/LastDocsComponent/LastDocs.jsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import Link from '@mui/material/Link';
+import Table from '@mui/material/Table';
+import TableBody from '@mui/material/TableBody';
+import TableCell from '@mui/material/TableCell';
+import TableRow from '@mui/material/TableRow';
+import Title from '../DashboardComponent/Title';
+import Typography from '@mui/material/Typography';
+
+function preventDefault(event) {
+    event.preventDefault();
+}
+class LastDocs extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            items: [],
+            checked: false
+        };
+        
+    }
+    componentDidMount() {
+        fetch('https://fakestoreapi.com/products?limit=5').then(res => res.json()).then(json => {
```

Comment:
componentDidMount is using in react with classes, that is, react legacy, in this case, we are using react with a hook if you need to control the fetch you need to put the request, into the useEffect function,